File: 0027-remove-element/solution.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int i=0,j,temp;
        for(j=0;j<nums.length;j++)
        {
            if(nums[j]!=val)
            {
            temp=nums[i];
            nums[i]=nums[j];
            nums[j]=temp;
            i++;
            }
        }
        return i;
    }
}

File: 0027-remove-element/solution.c
int removeElement(int* nums, int numsSize, int val) {
    int i,j;
    int k=numsSize;
    for(i=0;i<numsSize;i++)
    {
        if(nums[i]==val)
        {
            nums[i]=51;//nums value will be lesser than 50 only as per constraint
            k--;
        }
    }
    
    int temp;
    for(i=0;i<numsSize;i++)
    {
        for(j=i+1;j<numsSize;j++)
        {
            if(nums[i]>nums[j])
            {
                temp=nums[i];
                nums[i]=nums[j];
                nums[j]=temp;
            }
        }
    }
    return k;
}
